src/frequenz/sdk/microgrid/_power_wrapper.py

```diff
@@ -17,19 +16,13 @@
 from frequenz.client.microgrid import ComponentCategory, ComponentType
 
 from .._internal._channels import ChannelRegistry, ReceiverFetcher
-from . import _power_managing
-
-# A number of imports had to be done inside functions where they are used, to break
-# import cycles.
-#
-# pylint: disable=import-outside-toplevel
-if typing.TYPE_CHECKING:
-    from ..actor.power_distributing import (  # noqa: F401 (imports used by string type hints)
```

Comment:
Do we still have the "string type hints" or was the comment outdated?

Reply:
I don't think there were string type hints in that file.  My guess is this was copied over from somewhere when we added flake8.